examples/zkdetect/main.go

```diff
@@ -0,0 +1,53 @@
+package main
```

Comment:
> the go examples also run during unit testing. the zkdetect bin really isn't meant for that since it relies on an external zk process/cluster to be up and running.

OK. This makes sense.